### PR TITLE
Fixed and added chapter 0 from Intro branch

### PR DESCRIPTION
**Pull request by aarontaddiken**
[#95933724]

### CODE REVIEW

File: chapter_0/Readme.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Questions:
+
+Provide a link in your Readme.md that links to the documentation for the following classes:
+
+[Arrays](http://ruby-doc.org/core-2.2.0/Array.html)
+
+[Arrays](http://www.tutorialspoint.com/ruby/ruby_arrays.htm)
```
Comment (jasonnoble) [inline chapter_0/Readme.md]: Remove me

File: chapter_0/Readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+[Hash](http://ruby-doc.org/core-2.2.0/Hash.html)
+
+[Hash](http://www.gotealeaf.com/blog/how-the-hash-works-in-ruby)
```
Comment (jasonnoble) [inline chapter_0/Readme.md]: Remove me

File: chapter_0/Readme.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+[Chronic](http://ruby-doc.org/core-2.2.0/chronic.html)
+
+[More Chronic](http://chronic.rubyforge.org/)
```
Comment (jasonnoble) [inline chapter_0/Readme.md]: Remove me

File: chapter_0/Readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+[...getting Chronic](https://rubygems.org/gems/chronic_duration/versions/0.10.6)
```
Comment (jasonnoble) [inline chapter_0/Readme.md]: Remove me

File: chapter_0/Readme.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Find two other relative links and add those to the Readme.md as well.
```
Comment (jasonnoble) [inline chapter_0/Readme.md]: Find two other Ruby related documentation links.  Go to http://ruby-doc.org/core-2.2.0/ and find two other classes that interest you, link to them in the Readme.

**Comment by aarontaddiken** [comment]
Fixed Readme adding two more Class links and ridding extra links.

**Pull request merged**